src/lib/tokens.js

```diff
@@ -1,45 +1,6762 @@
+  'yah',
+  'yak',
+  'yale',
+  'yalta',
+  'yam',
+  'yamaha',
+  'yang',
+  'yank',
+  'yap',
+  'yaqui',
+  'yard',
+  'yarn',
+  'yates',
+  'yaw',
+  'yawl',
+  'yawn',
+  'yea',
+  'yeah',
+  'year',
+  'yearn',
+  'yeast',
+  'yeasty',
+  'yeats',
+  'yell',
+  'yelp',
+  'yemen',
+  'yen',
+  'yet',
+  'yield',
+  'yin',
+  'yip',
+  'ymca',
+  'yodel',
+  'yoder',
+  'yoga',
+  'yogi',
+  'yoke',
+  'yokel',
+  'yolk',
+  'yon',
+  'yond',
+  'yore',
+  'york',
+  'yost',
+  'you',
+  'young',
+  'your',
+  'youth',
+  'yow',
+  'yucca',
+  'yuck',
+  'yuh',
+  'yuki',
+  'yukon',
+  'yule',
+  'yves',
+  'ywca',
+  'yyy',
+  'yyyy',
+  'zag',
+  'zaire',
+  'zan',
+  'zap',
+  'zazen',
+  'zeal',
+  'zealot',
+  'zebra',
+  'zeiss',
+  'zen',
+  'zero',
+  'zest',
+  'zesty',
+  'zeta',
+  'zeus',
+  'zig',
+  'zilch',
+  'zinc',
+  'zing',
+  'zion',
+  'zip',
+  'zloty',
+  'zoe',
+  'zomba',
+  'zone',
+  'zoo',
+  'zoom',
+  'zorn',
+  'zurich',
+  'zzz',
+  'zzzz',
+];
 
 // the below is heavily based on https://andrew.hedges.name/experiments/diceware/
 
 // from: https://www.rempe.us/diceware/
 function secureRandom(count) {
-    var cryptoObj = window.crypto || window.msCrypto
-    var rand = new Uint32Array(1)
-    var skip = 0x7fffffff - 0x7fffffff % count
-    var result
-
-    if (((count - 1) & count) === 0) {
-        cryptoObj.getRandomValues(rand)
-        return rand[0] & (count - 1)
-    }
-
-    do {
-        cryptoObj.getRandomValues(rand)
-        result = rand[0] & 0x7fffffff
-    } while (result >= skip)
-
-    return result % count
+  var cryptoObj = window.crypto || window.msCrypto;
+  var rand = new Uint32Array(1);
+  var skip = 0x7fffffff - (0x7fffffff % count);
+  var result;
+
+  if (((count - 1) & count) === 0) {
+    cryptoObj.getRandomValues(rand);
+    return rand[0] & (count - 1);
+  }
+
+  do {
+    cryptoObj.getRandomValues(rand);
+    result = rand[0] & 0x7fffffff;
+  } while (result >= skip);
+
+  return result % count;
 }
 
 // based on: http://stackoverflow.com/a/1527820/11577
 function getRandomInt(min, max) {
-    if (window.crypto || window.msCrypto) {
-        return secureRandom(max) + min
-    }
-    else {
-        return Math.floor(Math.random() * (max - min + 1)) + min
-    }
+  if (window.crypto || window.msCrypto) {
+    return secureRandom(max) + min;
+  } else {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
 }
 
 function getToken() {
-    var phrase = []
+  var phrase = [];
+
+  for (var i = 0; i < 3; i += 1) {
+    phrase.push(words[getRandomInt(0, words.length)]);
+  }
 
-    for (var i = 0; i < 3; i += 1) {
-        phrase.push(words[getRandomInt(0, words.length)]);
-    }
+  return phrase.join(' ');
+}
+
+export function getListToken() {
+  return localStorage.getItem('token');
+}
 
-    return phrase.join(' ')
+//setListToken here
+export function generateListToken() {
+  return localStorage.setItem('token', getToken());
 }
 
 export default getToken;
```

Comment:
I like the decision to put the localstorage functionality in this file! This isn't a blocker, but now that you're not using `getToken` elsewhere, you could consider removing this export.